### PR TITLE
#6358 include missing plugins in pluginsConfig

### PR DESCRIPTION
**Pull request by MV88**
## Description

Added some of the missing plugins to pluginsConfig.json 

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [x] Feature - config update
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#6358

**What is the new behavior?**

those plugins now appear
image

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by offtherailz** [comment]
I think they should be orgainzed like this:
- [ ] `ExportDataResults` should be hidden and a dependency of `LayerDownload` (hope to merge them soon)
- [ ] `SearchByBookmark` maybe children of `SearchBar`
- [ ] `Swipe` should be children of `TOC`
What do you think matteo?

**Review by offtherailz — COMMENTED** [review]
It should work, but I think they should be orgainzed like this:
- [ ] `ExportDataResults` should be hidden and a dependency of `LayerDownload` (hope to merge them soon)
- [ ] `SearchByBookmark` maybe children of `SearchBar`
- [ ] `Swipe` should be children of `TOC`
What do you think @MV88 ?

**Comment by MV88** [comment]
> It should work, but I think they should be orgainzed like this:
> 
> * [ ]  `ExportDataResults` should be hidden and a dependency of `LayerDownload` (hope to merge them soon)
> * [ ]  `SearchByBookmark` maybe children of `SearchBar`
> * [ ]  `Swipe` should be children of `TOC`
>   What do you think @MV88 ?

thanks, i'll have a look

**Review by offtherailz — CHANGES_REQUESTED** [review]
the 2 pluginsConfig should be syncronized (the one in template and the one in web/client/

File: project/standard/templates/pluginsConfig.json [hunk 1]
```diff
@@ -29,6 +29,15 @@
...
     },
+    {
+      "name": "Swipe"
+    },
+    {
+      "name": "ExportDataResults"
+    },
+    {
+      "name": "SearchByBookmark"
+    },
```
Comment (offtherailz) [inline project/standard/templates/pluginsConfig.json:40]: I think this remained there in error, isn't it?

**Review by offtherailz — APPROVED** [review]

**Comment by offtherailz** [comment]
Thank you, please backport this to the stable branch

**Comment by ElenaGallo** [comment]
@MV88 the PR passed the test on dev, please backport this to the stable branch.

**Pull request merged**